.github/workflows/check.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
 
     env:
-      REACT_APP_BEE_HOSTS: 'https://bee-0.gateway.ethswarm.org/,https://bee-1.gateway.ethswarm.org/,https://bee-2.gateway.ethswarm.org/,https://bee-3.gateway.ethswarm.org/,https://bee-4.gateway.ethswarm.org/,https://bee-5.gateway.ethswarm.org/,https://bee-6.gateway.ethswarm.org/,https://bee-7.gateway.ethswarm.org/,https://bee-8.gateway.ethswarm.org/,https://bee-9.gateway.ethswarm.org/'
+      REACT_APP_BEE_HOSTS: 'https://api.gateway.ethswarm.org/'
```

Comment:
These should still be exposed, not sure why it does not work but will check with devops. Using the `api.gateway.ethswarm` means we lose the frontend optimization. See #218